poller/poller.go

```diff
@@ -104,3 +144,38 @@ func (poller *Poller) jobs() {
 		}
 	}
 }
+
+func generateSerializedData(splitData SplitData) (string, error) {
```

Comment:
can we add a description for this function to differentiate with `GetSerializedData `